app/models/entry.rb

```diff
@@ -31,6 +31,6 @@ def self.top_by_user
 
   # Returns true is an entry was created by given user
   def owned_by?(current_user)
-    user == current_user
+    current_user && user_id == current_user.id
```

Comment:
Where is `user_id` coming from?

Reply:
It comes from https://github.com/igrigorik/vimgolf/blob/master/app/models/entry.rb#L18
```
belongs_to :user
```
belongs to automatically create `relation_id` https://docs.mongodb.com/mongoid/master/tutorials/mongoid-relations/#belongs-to


I didn't precise : We get all these User.find on Challenge#show : one User.find per printed entry

Reply:
Gotcha, thanks!